localstack/services/kms/models.py

```diff
@@ -646,6 +653,15 @@ def get_key_id_from_any_id(
                 raise NotFoundException(f"Unable to find KMS alias with name {alias_name}")
             key_id = self.aliases[alias_name].metadata["TargetKeyId"]
 
+        # regular KeyId are UUID, and MultiRegion keys starts with 'mrk-' and 32 hex chars
+        if not PATTERN_UUID.match(key_id) and not MULTI_REGION_PATTERN.match(key_id):
```

Comment:
**nit:**
You could use the UUID module here to check validity instead of a regex - it is probably safer.
Also could use simple string operations instead of regex in the multi region pattern, but I think there it is ok.

Reply:
I didn't know about that!
I try looking around for solution and apparently it's not always valid?
https://stackoverflow.com/questions/53847404/how-to-check-uuid-validity-in-python
https://gist.github.com/ShawnMilo/7777304

And it needs a try/except clause for that, not sure if we should change it?

Reply:
Maybe we can add a utility function for that? Seems like something which is useful in more places. The utils use quite a lot of try/except, so it wouldn't look too out of place there :)

Reply:
I'll need a test case for that, because Python will accept more than this regex:
With the try/except clause:
```python
>>> is_valid_uuid("{20f5484b-88ae-49b0-8af0-3a389b4917dd}")
True
>>> is_valid_uuid("20f5484b88ae49b08af03a389b4917dd")
True
```
I'm not sure AWS will accept those things.

Reply:
I added a test case and what would be a valid UUID for Python is not for AWS, so we cannot really use the module for this particular use case. Thanks for pointing it though, it would have been cleaner and better.